lib/PuppeteerSharp.Tests/WaitTaskTests/FrameWaitForFunctionTests.cs

```diff
@@ -216,5 +232,59 @@ public async Task ShouldSurviveNavigations()
             await Page.EvaluateFunctionAsync("() => window.__done = true");
             await watchdog;
         }
+
+        private TaskCompletionSource<bool> ListenForStartPolling()
+        {
+            TaskCompletionSource<bool> startedPolling = new TaskCompletionSource<bool>();
+
+            // Wait for function will release the execution faster than in node.
+            // We intercept the poller.start() call to prevent tests from continuing before the polling has started.
+            _connectionTransportInterceptor.MessageSent += (_, message) =>
+            {
+                if (message.Contains("poller => poller.start()"))
```

Comment:
Let's add a comment where we are executing the `poller => poller.start()`. Mentioning that if we change the code there we need to update this test.